src/main/java/com/bankservice/bankservice/repository/AccountRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.bankservice.bankservice.repository;
+
+import com.bankservice.bankservice.model.Account;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface AccountRepository extends JpaRepository<Account, Long> {
+    Optional<Account> findAccountByAccountNumber(String accountNumber);
```

Comment:
```suggestion
    Optional<Account> findByAccountNumber(String accountNumber);
```
Will it work with this method name?